src/server/api/schema.js

```diff
@@ -322,7 +323,7 @@ async function editCampaign(id, campaign, loaders, user) {
 
 const rootMutations = {
   RootMutation: {
-    sendReply: async (_, { id, message }, { loaders }) => {
+    sendReply: async (_, { id, message }, { user, loaders }) => {
       if (process.env.NODE_ENV !== 'development') {
```

Comment:
I imagine we'll want to use this to test in a production-like environment - so we'd have to deploy it in dev mode to do that?

Reply:
ah, i missed that.  i'm going to make is so development node env OR have correct access